controlplane/rosa/api/v1beta2/rosacontrolplane_types.go

```diff
@@ -95,12 +95,14 @@ type RosaControlPlaneSpec struct { //nolint: maligned
 
 	// The Subnet IDs to use when installing the cluster.
 	// SubnetIDs should come in pairs; two per availability zone, one private and one public.
-	Subnets []string `json:"subnets"`
+	// +optional
```

Comment:
rosaControlPlane-webhook validation is required either Subnets & AvailabilityZones exist OR ROSANetworkRef